asset/synth.py

```diff
@@ -85,3 +78,9 @@
     _BORKED_ASSET_DOCSTRING,
     _FIXED_ASSET_DOCSTRING,
 )
+
+# ----------------------------------------------------------------------------
+# Add templated files
+# ----------------------------------------------------------------------------
+templated_files = gcp.CommonTemplates().py_library(unit_cov_level=86, cov_level=85)
```

Comment:
For now that is likely the case. Really this documents where we are. I, along with Thea, have some thoughts on better isolating us from generated code for these coverage stats. But sadly for now it may be a small bit of tweaking here and there. proto and gapic dirs are ignored so it cuts down on most noise.